salesforce_api/gsheet-new-row-to-sfdc-new-contact/Readme.md

```diff
@@ -0,0 +1,64 @@
+Example to create new contacts in the Salesforce using Google Sheets and Salesforce integration.
```

Comment:
Link to salesforce and google sheets

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)